setup.cfg

```diff
@@ -86,6 +86,7 @@ install_requires =
     # cattrs >= 1.1.0 dropped support for Python 3.6
     cattrs>=1.0, <1.1.0;python_version<="3.6"
     cattrs~=1.1;python_version>"3.6"
+    chardet<4.0.0
```

Comment:
chardet is a common dependency across many providers and 4.0.0 is conflicting with several of them.

Reply:
I suggest removing it from here if not required by core even if needed by several providers

Reply:
Yep. I will build the image and try to analyse it deeper to find out where to add them (in providers)

Reply:
This is another problem with snowflake, if I remember correctly.